ui/raidboss/data/06-ew/dungeon/the_excitatron_6000.ts

```diff
@@ -0,0 +1,148 @@
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+// TODO: id for Lucky Face Temper's Flare
+// TODO: all Lucky Sphinx abilities
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.TheExcitatron6000,
+
+  triggers: [
+    // ---------------- random treasure mobs ----------------
+    {
+      id: 'Excitatron Rainbow Golem Spawn',
+      type: 'AddedCombatant',
+      netRegex: { name: 'Rainbow Golem', capture: false },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Rainbow Golem spawned!',
+        },
+      },
+    },
+    {
+      id: 'Excitatron Golden Supporter Spawn',
+      type: 'AddedCombatant',
+      netRegex: { name: 'Golden Supporter', capture: false },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Golden Supporter spawned!',
+        },
+      },
+    },
+    {
+      id: 'Excitatron Exciting Mandragoras Spawn',
+      type: 'AddedCombatant',
+      netRegex: {
+        name: [
+          'Exciting Tomato',
```

Comment:
Can you use `npcNameId` fields for these instead of translatable names?

Reply:
done.